GLTFSDK/Source/MeshPrimitiveUtils.cpp

```diff
@@ -524,23 +524,19 @@ namespace
             throw GLTFException("Input triangulated triangle strip has fewer than 3 indices.");
         }
 
-        const size_t triangleCount = indexCount / 3;
-        const auto resultIndexCount = 2 + triangleCount;
-        std::vector<T> result(resultIndexCount);
+        std::vector<T> result;
```

Comment:
You need to call reserve on this now that we're using push_back.

Reply:
Done.